docs/scenarios/parametric.md

```diff
@@ -202,41 +222,22 @@ There is two ways for running the C++ library tests with a custom tracer:
     * `https://github.com/DataDog/dd-trace-cpp@<COMMIT HASH>`
 2. Clone the dd-trace-cpp repo inside `binaries`
 
-The parametric shared tests can be run against the C++ library,
-[dd-trace-cpp][1], this way:
-```console
-$ TEST_LIBRARY=cpp ./run.sh PARAMETRIC
-```
-
-Use the `-k` command line argument, which is forwarded to [pytest][2], to
-specify a substring within a particular test file, class, or method. Then only
-matching tests will run, e.g.
-```console
-$ TEST_LIBRARY=cpp ./run.sh PARAMETRIC -k test_headers
-```
-
-It's convenient to have a pretty printer for the tests' XML output. I use
-[xunit-viewer][3].
-```console
-$ npm install junit-viewer -g
-```
-
-My development iterations then involve running the following at the top of the
-repository:
-```console
-$ TEST_LIBRARY=cpp ./run.sh PARAMETRIC -k test_headers; xunit-viewer -r logs_parametric/reportJunit.xml
+#### When you are done testing against a custom tracer:
```

Comment:
Let's link to this section at the top of the `### Running the tests for a custom tracer` section so that new users (the ones most likely to get caught off guard) have a higher chance of seeing this. Something like "note that any modification to the binaries will change the version of the tracer used across your local computer. After you're done testing with the custom version, be sure to remove it from the binaries by following <link>". 

I think you don't need to add an example per tracer here but one or two would be helpful along with just an explanation that you should remove it from the binaries if that's how you added it.